rainbowriders.p8

```diff
@@ -54,6 +54,57 @@ INDIGO = 13
 PINK = 14
 PEACH = 15
 
+-- A collection of special effects that are easy to use
+-- This is a factory function that returns an object with special effects
+function SpecialEffect ()
```

Comment:
This is awesome, we may want to look into breaking this out into a utility lua script and just include it in all carts going forward.

https://www.lexaloffle.com/dl/docs/pico-8_manual.html#_INCLUDE